### PR TITLE
Use store path as hover title for store badge

### PR DESCRIPTION
**Pull request by erayd**
When hovering over the store name badge in multi-store mode, display the path to that store.

scr-20180403-150730

### CODE REVIEW

File: chrome/script.browserify.js [hunk 1]
```diff
@@ -49,11 +49,18 @@ function view() {
         };
 
         var store = "default";
+        var storeTitle = "Default password store";
         var name = login;
         var i;
         if ((i = login.indexOf(":"))) {
           if (searchSettings && searchSettings.customStores.length > 1) {
             store = login.substr(0, i);
+            searchSettings.customStores.map(function(customStore) {
```
Comment (maximbaz) [inline chrome/script.browserify.js]: unusual use of `map` 🙂

Comment (erayd) [reply chrome/script.browserify.js]: Oops - that was a mistake. Intention was `forEach`. I think I must have been looking at `map` elsewhere, and typed it accidentally.

No real functional impact using it here though - so we can leave it as-is - but would you like me to change it for clarity?

Comment (erayd) [reply chrome/script.browserify.js]: Or I guess I could have used a `(for i in searchSettings.customStores) {` loop too.

Comment (maximbaz) [reply chrome/script.browserify.js]: I'll maybe change to `forEach` next time I need to do some changes in that file, as you said there is no functional impact anyway 😉

**Pull request merged**